### PR TITLE
Add bitwarden

### PR DESCRIPTION
**Pull request by OmarNamis**

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
Is it just getting into the link and hitting a delete button?

File: _data/sites.json [hunk 1]
```diff
@@ -934,6 +934,15 @@
...
 
+    {
+        "name": "Bitwarden",
+        "url": "https://bitwarden.com/help/article/delete-your-account",
+        "difficulty": "easy",
+        "domains": [
+            "bitwarden.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:945]: Can you move it to be above `Blinkist` to keep the list sorted? 

Thanks!

**Comment by OmarNamis** [comment]
> 
> 
> Is it just getting into the link and hitting a delete button?

There are 2 ways:

1- If you know the password, log in and open account settings. There you'll find a delete button that deletes directly without sending a confirmation email.

2- If you don't remember the password, go to https://vault.bitwarden.com/#/recover-delete and enter your email.
You will receive a confirmation email with a link to delete your account.

Should I add this info as a note? It's already described in the link that I added.

File: _data/sites.json (hunk 1 above)
Comment (OmarNamis) [reply _data/sites.json:945]: Oops! I must've missed it.

**Comment by tupaschoal** [comment]
> Should I add this info as a note? It's already described in the link that I added.

No, if it's all in the link, no need to create redundancy

File: _data/sites.json (hunk 1 above)
Comment (tupaschoal) [reply _data/sites.json]: Don't worry, the CI didn't kick in so you really couldn't know.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**